### DISCUSSION TITLE
Data for Singapore missing since May 15, 2022.

### DISCUSSION THREAD

**Discussion by fehrevenue** [owid/covid-19-data#2659] [General]
The infection data for Singapore is not included in the report since 15 May 2022.  The Singapore Ministry of Health is still publishing the data on their website(www.moh.gov.sg) Is there any reason for the missing data?

![thumbnail_image](https://user-images.githubusercontent.com/51076064/169727779-cfc1541b-17cd-4c67-a3f0-42add39d9966.png)

**Comment by lucasrodes** [comment]
Hi @fehrevenue,
Thanks for your report.

We source our data on COVID-19 confirmed cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University: https://github.com/CSSEGISandData/COVID-19.
This page also lists their sources, country by country.
We report this data without alterations, but issues can be raised in their GitHub repository: https://github.com/CSSEGISandData/COVID-19/issues

**Reply by fehrevenue** [reply]
Thanks.  I have raised the issue on the JHU github.
